.github/workflows/lint.yml

```diff
@@ -0,0 +1,39 @@
+name: Lint
+
+on:
+  pull_request:
+
+jobs:
+  rustfmt:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v4
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly
```

Comment:
Would you happen to have any reason to use the nightly and not the stable toolchain?